src/api/parser/src/utils/supabase.js

```diff
@@ -0,0 +1,21 @@
+const { logger } = require('@senecacdot/satellite');
+const { createClient } = require('@supabase/supabase-js');
+
+const { SERVICE_ROLE_KEY, SUPABASE_URL } = process.env;
+
+const supabase = createClient(SUPABASE_URL, SERVICE_ROLE_KEY);
```

Comment:
Use the admin `SERVICE_ROLE_KEY` only when you need to, it can bypass any RLS.
```suggestion
const supabase = createClient(SUPABASE_URL, ANON_KEY);
```